src/System.Memory/src/System.Memory.csproj

```diff
@@ -5,6 +5,9 @@
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <Configurations>netcoreapp-Unix-Debug;netcoreapp-Unix-Release;netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;netcoreappaot-Windows_NT-Debug;netcoreappaot-Windows_NT-Release;uap-Windows_NT-Debug;uap-Windows_NT-Release</Configurations>
   </PropertyGroup>
+  <ItemGroup>
+    <GenFacadesIgnoreMissingType Include="System.Buffers.SequenceReader`1" />
```

Comment:
For the GenFacadesOmitType  and GenFacadesIgnoreMissingType  is a comment possible to explain "why" (in case you and @ericstj aren't available to ask)

Reply:
Actually I don’t understand why we are suppressing this.

Reply:
OK, then in case @anipik isn't around to ask 😆

Reply:
Yeah i will add the comment.
We are using 2.0.0 version of Microsoft.CodeAnalysis.CSharp. This particular version has a bug where it is not able to parse ```public partial ref struct```  properly. It drops the public modifer.
We cant change the version of the assembly due to reason described here https://github.com/dotnet/arcade/pull/2221/files#diff-77a8f3b81eed042aa8a317f278b1037bR18

Reply:
Ah, now I remember.  We should file an issue on this and reference the issue from the comment.  It should go away once we can upgrade to the 3.0 CLI.

Reply:
Is this going to break folks dogfooding latest VS where msbuild loads newer Microsoft.CodeAnalysis.CSharp ? Should this rather launch in a separate process?

Reply:
@Anipik that's a good test case, can you try it out and make sure you don't have duplicate forwards?

Out of proc is possible but it feels pretty heavy handed just to workaround the MSBuild unification bug.